services/admin/jest.config.js

```diff
@@ -15,10 +15,10 @@ module.exports = {
   ],
   coverageThreshold: {
     global: {
-      statements: 90,
-      branches: 90,
-      functions: 90,
-      lines: 90,
+      statements: 100,
+      branches: 100,
+      functions: 100,
+      lines: 100,
```

Comment:
This is the default from the shared config, so the whole `coverageThreshold` part can be deleted.